crates/sui-core/src/state_accumulator.rs

```diff
@@ -431,19 +427,12 @@ impl StateAccumulator {
             .collect();
 
         if !remaining_checkpoints.is_empty() {
-            debug!(
-                "Awaiting accumulation of checkpoints {:?} for epoch {} accumulation",
+            panic!(
+                "Checkpoints {:?} expected to already be accumulated during epoch {} accumulation.",
                 remaining_checkpoints, epoch
```

Comment:
nit: consider renaming the variable remaining_checkpoints as it sounds like it's normal.